src/sx/src/__tests__/transformValue.test.js

```diff
@@ -30,3 +30,19 @@ test.each`
     expect(transformValue(styleName, styleValue)).toBe(expectedValue);
   },
 );
+
+test.each`
+  rawValue     | normalizedValue
+  ${'0.1'}     | ${'.1'}
+  ${'1.1'}     | ${'1.1'}
+  ${'0.75rem'} | ${'.75rem'}
+  ${'.75rem'}  | ${'.75rem'}
+  ${'75rem'}   | ${'75rem'}
+  ${'4px'}     | ${'4px'}
+  ${'.4px'}    | ${'.4px'}
+  ${'0.4px'}   | ${'.4px'}
+  ${10}        | ${'10px'}
+  ${0.1}       | ${'.1px'}
```

Comment:
I'd one more test to be sure it works as expected:
```js
${'0px'} | ${'0px'}
```

Reply:
fixed 👍